sympy/matrices/common.py

```diff
@@ -732,6 +732,8 @@ def size(m):
             """Compute the size of the diagonal block"""
             if hasattr(m, 'rows'):
                 return m.rows, m.cols
+            if isinstance(m,dict):
```

Comment:
Looking good.

Please do PEP8 tidy (e.g. spaces after commas, space around `+-` operators, comparisons like ` == `, and after `if`)